README.md

```diff
@@ -10,6 +10,27 @@ Data Model Library - A collection of QBiC data models.
 ## Author
 Created by Andreas Friedrich, Luis de la Garza, Sven Fillinger.
 
+Overview:
+
+- [How to Install](#how-to-install)
+- [Data Models](#data-models)
+    * [Oxford Nanopore Data Structure](#oxford-nanopore-data-structure)
+    
+        * [Nanopore result set for registration at QBiC](#nanopore-result-set-for-registration-at-qbic)
+        * [Nanopore usage example](#nanopore-usage-example)
+        * [Nanopore data structures translated in openBIS](#nanopore-data-structures-translated-in-openbis)
+        
+    * [NF-Core Bioinformatics Analysis Result Sets](#nf-core-bioinformatics-analysis-result-sets)
+    
+        * [NF-Core data structure for registration at QBiC](#nf-core-data-structure-for-registration-at-qbic)
+        * [NF-Core usage example](#nf-core-usage-example)
+        * [NF-Core data structures translated in openBIS](#nf-core-data-structures-translated-in-openbis)
+        
```

Comment:
I think the repetition of Nanopore and NF-Core is not necessary as they are subpoints of a point where the tile is already mentioned.